Avoid "whitelist" according to rust-lang/rust changes (#821)
rust-lang/rustc-dev-guide: src/stability.md

Macros, compiler desugarings and `const fn`s expose their bodies to the call
site. To work around not being able to use unstable things in the standard
library's macros, there's the `#[allow_internal_unstable(feature1, feature2)]`

> attribute that allows the given features for usage in stable macros or

Reviewer: ```suggestion
attribute that allows the given features to be used in stable macros or
```
